pipeline/README.md

```diff
@@ -1,27 +1,9 @@
 # Illinois NLP Pipeline
```

Comment:
The changes here are mostly re-ordering. Lots of content seemed to be out of place, since they were written in different times.